app/views/group_offers/_form.html.slim

```diff
@@ -26,19 +26,9 @@
 
   .row
     .col-xs-12.col-md-4
-      = f.input :necessary_volunteers, collection: 1..10
-    .col-xs-12.col-md-8
-      - if policy(GroupOffer).supervisor_privileges?
-        = f.association :volunteers
-      - else
-        = f.association :volunteers,
-          collection: Volunteer.without_active_assignment.not_in_any_group_offer.not_responsible
-
-  .row
-    .col-xs-12.col-md-4
-      - if policy(GroupOffer).supervisor_privileges?
+      - if policy(GroupOffer).superadmin?
```

Comment:
`.superadmin?` and `department_manager?` policies are not defined. To me there is no reason to change the `superadmin_priviledges?` policy. Also there is no reason to have to ifs executed when only one of the cases will be true each time.